roles/StackStorm.redis/tasks/redis_redhat.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Install redis
+  become: yes
+  package:
+    name: redis
```

Comment:
I see the package naming is different between the el/deb.
If that's the only case, maybe we could do it via vars instead and keep the package installation in a single main file?
https://github.com/StackStorm/ansible-st2/blob/b15ea27511ccf579809c2d5ceb0fe3e6f6fbfda3/roles/StackStorm.mongodb/tasks/main.yml#L2-L6